jdaviz/core/template_mixin.py

```diff
@@ -1018,8 +1021,27 @@ class DatasetSelect(BaseSelectPluginComponent):
     """
     def __init__(self, plugin, items, selected,
                  filters=['not_from_plugin_model_fitting', 'layer_in_viewers'],
+                 default_text=None, manual_options=[],
```

Comment:
We didn't have a use for manual options in dataset dropdowns until now, so they weren't exposed.  This works the same way as all other Select components that inherit from BaseSelectPluginComponent.